api/scpca_portal/models/computed_file.py

```diff
@@ -194,223 +239,6 @@ def get_project_file(cls, project, download_config: Dict, computed_file_name: st
-        cls, project, sample_to_file_mapping, workflow_versions, file_format
-    ):
-        """Prepares a ready for saving single data file of project's merged data."""
-
-        project_file_mapping = {
-            project.input_merged_summary_report_file_path: (
-                f"{project.scpca_id}_merged-summary-report.html"
-            ),
-        }
-
-        if file_format == cls.OutputFileFormats.ANN_DATA:
-            if not project.includes_merged_anndata:
-                return None
-            computed_file_name = project.output_merged_anndata_computed_file_name
-            readme_file_path = ComputedFile.README_ANNDATA_MERGED_FILE_PATH
-            project_file_mapping[
-                f"{project.input_merged_data_path}/{project.scpca_id}_merged_rna.h5ad"
-            ] = f"{project.scpca_id}_merged_rna.h5ad"
-
-            if project.has_cite_seq_data:
-                project_file_mapping[
-                    f"{project.input_merged_data_path}/{project.scpca_id}_merged_adt.h5ad"
-                ] = f"{project.scpca_id}_merged_adt.h5ad"
-        else:
-            if not project.includes_merged_sce:
-                return None
-            computed_file_name = project.output_merged_computed_file_name
-            readme_file_path = ComputedFile.README_SINGLE_CELL_MERGED_FILE_PATH
-
-            project_file_mapping[
-                f"{project.input_merged_data_path}/{project.scpca_id}_merged.rds"
-            ] = f"{project.scpca_id}_merged.rds"
-
-        computed_file = cls(
-            format=file_format,
-            includes_merged=True,
-            modality=cls.OutputFileModalities.SINGLE_CELL,
-            project=project,
-            s3_bucket=settings.AWS_S3_BUCKET_NAME,
-            s3_key=computed_file_name,
-            workflow_version=utils.join_workflow_versions(workflow_versions),
-        )
-
-        project_file_mapping[readme_file_path] = ComputedFile.OUTPUT_README_FILE_NAME
-        project_file_mapping[
-            project.output_single_cell_metadata_file_path
-        ] = computed_file.metadata_file_name
-
-        if project.has_bulk_rna_seq:
-            project_file_mapping[project.input_bulk_metadata_file_path] = "bulk_metadata.tsv"
-            project_file_mapping[project.input_bulk_quant_file_path] = "bulk_quant.tsv"
-
-        sample_file_suffixes = {
-            "celltype-report.html",
-            "qc.html",
-        }
-
-        with ZipFile(computed_file.zip_file_path, "w") as zip_file:
-            for src, dst in project_file_mapping.items():
-                zip_file.write(src, dst)
-
-            for sample_id, file_paths in sample_to_file_mapping.items():
-                for file_path in file_paths:
-                    if str(file_path).split("_")[-1] not in sample_file_suffixes:
-                        continue
-                    # Nest these under their sample id.
-                    zip_file.write(file_path, Path("individual_reports", sample_id, file_path.name))
```

Comment:
@davidsmejia Also, I wanted to ask about what this nesting of non-merged library reports inside a `individual_reports` directory during merged file computation was all about. I believe I should update my PR to also accommodate this nested directory.

Reply:
Yeah good catch, it should match whatever we say we are doing in terms of file structure.